docs/docs-new/pages/product/configuration/data-sources/duckdb.mdx

```diff
@@ -0,0 +1,115 @@
+---
+redirect_from:
+  - /config/databases/aws-athena
+---
+
+# DuckDB
+
+[duckdb] is an in-process SQL OLAP database management system, and has support
+for querying data in CSV, JSON and Parquet formats from an AWS S3 bucket.
```

Comment:
DuckDB actually works with [S3, GCS, and Cloudflare R2](https://duckdb.org/docs/guides/import/s3_import#how-to-load-a-parquet-file-directly-from-s3-gcs-or-r2). Let's make this docs page less S3-centric.